numpy/core/numeric.py

```diff
@@ -3007,6 +3010,12 @@ class errstate(object):
     that context to execute with a known error handling behavior. Upon entering
     the context the error handling is set with `seterr` and `seterrcall`, and
     upon exiting it is reset to what it was before.
+    
+    ..  versionchanged: 1.14.0
+        In Python 3, errstate is also usable as a function decorator, saving
```

Comment:
Suspect you might need a blank line here, or that the following section needn't be indented - but I'm not at all sure